app/src/main/java/org/jellyfin/mobile/player/interaction/PlayerNotificationHelper.kt

```diff
@@ -123,7 +123,11 @@ class PlayerNotificationHelper(private val viewModel: PlayerViewModel) : KoinCom
             for (notificationAction in PlayerNotificationAction.values()) {
                 filter.addAction(notificationAction.action)
             }
-            context.registerReceiver(notificationActionReceiver, filter)
+            if (AndroidVersion.isAtLeastT) {
+                context.registerReceiver(notificationActionReceiver, filter, Context.RECEIVER_NOT_EXPORTED)
+            } else {
+                context.registerReceiver(notificationActionReceiver, filter)
+            }
```

Comment:
I believe there is a ContextCompat function we can use here instead

Reply:
Yes, good point. Gonna swap that one out.